packages/ove-service-imagetiles/README.md

```diff
@@ -0,0 +1,31 @@
+# Image Tile Service
+
+This service uses the [LibVips](https://jcupitt.github.io/libvips/) library via [NetVips](https://github.com/kleisauke/net-vips) to take Image based Assets and create [Deep Zoom](https://en.wikipedia.org/wiki/Deep_Zoom) image tile sets.

```

Comment:
```suggestion
This service uses the [LibVips](https://jcupitt.github.io/libvips/) library (via the [NetVips](https://github.com/kleisauke/net-vips) bindings) to create [Deep Zoom](https://en.wikipedia.org/wiki/Deep_Zoom) image tile sets from image files.
```

Reply:
clarified to note that this processes Assets